utils/odos.ts

```diff
@@ -0,0 +1,17 @@
+export async function fetchOdosApi<T>(
```

Comment:
This is a matter of preference but what do you think about putting this code in `api/odos.ts`. It's not really a utility used anywhere else.